### PR TITLE
Create/accept upgrade v3/v4 proposals

### PR DESCRIPTION
**Pull request by shayne-fletcher**
Add code to `davl-admin-cli`:
 - create v3 upgrade v4 proposals for all employee roles;
 - exercise accept on those proposals for employees indicated by `test-setup.json`.

### CODE REVIEW

File: admin-cli/src/index.ts [hunk 1]
```diff
@@ -30,36 +30,92 @@ function connect(config: Config, party: string): Ledger {
...
+      console.log(`Created UpgradeProposal for ${employee}.`);
+      if (config.employees[employee].acceptUpgrade) {
+        const employeeLedger = connect(config, employee);
+        await employeeLedger.exercise(
+          davlUpgradev3v4.UpgradeProposal.UpgradeProposal_Accept,
+          employeeProposalContract.contractId,
+          {},
+        );
+        console.log(`Accepted UpgradeProposal for ${employee}.`);
+      }
```
Comment (hurryabit) [inline admin-cli/src/index.ts]: I'd like to use this command with the DAVL we run in production. There's no config file for that one. Let's put something like accepting _all existing_ `UpgradeProposal`s in a separate command in a separate PR.

File: admin-cli/src/index.ts [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 import * as davl3 from '@daml2ts/davl-v3/lib/edb5e54da44bc80782890de3fc58edb5cc227a6b7e8c467536f8674b0bf4deb7/DAVL';
-// Check we can import the v3-v4 Upgrade module.
-import { } from '@daml2ts/davl-upgrade-v3-v4/lib/b31fe1021c80fcd4e0adc3437d24a328f3b721e81c0a158f6c4a94b89cb8ab32/Upgrade';
+import * as davlUpgradev3v4 from '@daml2ts/davl-upgrade-v3-v4/lib/b31fe1021c80fcd4e0adc3437d24a328f3b721e81c0a158f6c4a94b89cb8ab32/Upgrade';
 
 type EmployeeInfo = {
   boss: string;
   vacationDays: number;
   acceptProposal: boolean;
+  acceptUpgrade: boolean;
```
Comment (hurryabit) [inline admin-cli/src/index.ts]: Let's remove this for the reasons below.

File: admin-cli/src/index.ts [hunk 3]
```diff
@@ -30,36 +30,92 @@ function connect(config: Config, party: string): Ledger {
...
+}
+
+namespace v3v4 {
+  export async function upgrade (config: Config): Promise<void> {
+    const companyLedger = connect(config, config.company);
+    const employees = await companyLedger.query(davl3.EmployeeRole, {});
+    for (const employeeRole of employees) {
+      const employee = employeeRole.key;
+      const employeeProposal: davlUpgradev3v4.UpgradeProposal = {
+        employee:employee,
```
Comment (hurryabit) [inline admin-cli/src/index.ts]:
```suggestion
        employee: employee,
```
These ain't named arguments in OCaml. 😉

File: admin-cli/src/index.ts [hunk 4]
```diff
@@ -30,36 +30,92 @@ function connect(config: Config, party: string): Ledger {
...
+      await v3v4.upgrade(config);
+      break;
+    }
+    default: {
+      console.log ('Unrecognized command "' + process.argv[2] + '"');
+      usage();
+      break;
     }
   }
+
```
Comment (hurryabit) [inline admin-cli/src/index.ts]: Very important! 😉

File: test-setup.json [hunk 5]
```diff
@@ -8,22 +8,26 @@
     "Martin": {
       "boss": "Shaul",
       "vacationDays": 25,
-      "acceptProposal": false
```
Comment (hurryabit) [inline test-setup.json]: Let's not change anything here. See above why.

File: admin-cli/src/index.ts (hunk 1 above)
Comment (shayne-fletcher) [reply admin-cli/src/index.ts]: ✔️

File: admin-cli/src/index.ts (hunk 2 above)
Comment (shayne-fletcher) [reply admin-cli/src/index.ts]: ✔️

File: admin-cli/src/index.ts (hunk 3 above)
Comment (shayne-fletcher) [reply admin-cli/src/index.ts]: ✔️

File: test-setup.json (hunk 5 above)
Comment (shayne-fletcher) [reply test-setup.json]: ✔️

File: admin-cli/src/index.ts (hunk 4 above)
Comment (shayne-fletcher) [reply admin-cli/src/index.ts]: ✔️
